OpenRA.Mods.Common/Traits/AutoTarget.cs

```diff
@@ -24,6 +24,9 @@ public class AutoTargetInfo : ConditionalTraitInfo, Requires<AttackBaseInfo>, IE
 		[Desc("It will try to hunt down the enemy if it is set to AttackAnything.")]
 		public readonly bool AllowMovement = true;
 
+		[Desc("It will try to pivot to face the enemy if stance is not HoldFire.")]
+		public readonly bool AllowTurning = true;
```

Comment:
Do we have a reasonable use-case for disabling this? If not, it would be better to keep it hardcoded so that we don't need to worry about migrating it in the future if we rewrite the logic and want to remove it.

Reply:
The use-case I'm thinking of is to enable mods to add units with limited firing arcs that need to be explicitly ordered to reposition in the style of AT guns and MG crews in Company of Heroes.

Also a sense of symmetry with `AllowMovement` I guess.